src/test/regress/sql/with_clause.sql

```diff
@@ -390,3 +390,18 @@ with recursive rcte(x,y) as
   where t.b = x
 )
 select * from rcte limit 10;
+
+-- ensure orca does not fail when one cte on coordinator has correct flow and another - not
```

Comment:
I think it would be nice to point, that problem may be reproduced on build with assert only.

Reply:
done

Reply:
> As far as this test checks only orca behavior, maybe move it to gporca test file?

How? `gporca.sql` has `gporca.out` and `gporca_optimizer.out`